### ISSUE TITLE
Personal average pace caret

### ISSUE THREAD

**Issue by necdetaliozdur** [Miodec/monkeytype#370]
A pace caret that moves at personal average would be great for people who use monkey-type to gradually improve their typing skills. It would set a more reasonable goal than personal best.

**Comment by Miodec** [comment]
Not available at the moment but you can set the pace caret to a custom speed

**Comment by Jonny-exe** [comment]
Anyone working on this? I would like to do it. @Miodec @necdetaliozdur

**Comment by Miodec** [comment]
I believe nobody is working on it. What would be your approach to it?

**Comment by Jonny-exe** [comment]
I already implemented it.
I created a function called `db_getUserAverageWpm10` which I modeled after function `db_getUserHighestWpm`.
In `commandsPaceCaret` I added a new entry for average.
In `initPaceCaret()` I added an else portion to the if.

I tested everything locally, but for testing purpuses I set the average number to a hardcoded number.

I tried testing with firebase but when I try to register a user I get the following error back. `[ CONFIGURATION_NOT_FOUND ]`. How can I do testing with firebase?
